manifests.yaml

```diff
@@ -0,0 +1,38 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: powerdummy-populator
+  namespace: relationships
+spec:
+  replicas: 1
+  revisionHistoryLimit: 5
+  selector:
+    matchLabels:
+      app: powerdummy-populator
+  template:
+    metadata:
+      labels:
+        app: powerdummy-populator
+        team: relationships
+    spec:
+      containers:
+        - name: powerdummy-populator
+          image: eu.gcr.io/cognitedata/powerdummy-populator
+          resources:
+            requests:
+              memory: 300Mi
+              cpu: 200m
+            limits:
+              memory: 600Mi
+              cpu: 500m
+          env:
+            - name: COGNITE_BASE_URL
+              value: "https://greenfield.cognitedata.com"
+            - name: COGNITE_CLIENT_NAME
+              value: "powerdummy-populator"
+            - name: POWERDUMMY_API_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: powerdummy-apikey
+                  key: api-key
```

Comment:
add restartPolicy and backoffLimit see https://github.com/cognitedata/api/blob/master/api-smoke-test.yaml#L83

Reply:
fixed.. i assume "Never" and "0" are resonable here as well